python/src/retrain.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+RETRAINING_DIR = "/retraining"
+ARTIFACTS_DIR = f"{RETRAINING_DIR}/artifacts"
+METRICS_FILE = f"{RETRAINING_DIR}/metrics.json"
```

Comment:
can you also add the paths from which we need to read the training and serving data?

Reply:
Done